parsl/tests/test_monitoring/test_basic.py

```diff
@@ -120,7 +120,7 @@ def test_row_counts(tmpd_cwd, fresh_config):
             # Two entries: one showing manager active, one inactive
             result = connection.execute(text("SELECT COUNT(*) FROM node"))
             (c, ) = result.first()
-            assert c == 2
+            assert c == 3
```

Comment:
i'm a little surprised that this isn't 4 -- there's a message when a task is sent to a worker, but not when a result comes back. If this PR is to get more accurate "what's on my managers" info in the monitoring system, does that mean you don't see the reported load decrease properly?

Reply:
That's correct; I made this before I made the other PR, and I have the change locally to do just this.  But I didn't want to muddle #3834 after I put together this missing detail in my mind.  Since that's now merged, I'll this PR presently.

Reply:
Address in #3837; had meant to add it to this PR, but missed word in my previous message in this thread: _update_.  Heh.